projects/1.20-fabric/assets/paladins-and-priests/paladins/lang/zh_cn.json

```
@@ -0,0 +1,81 @@
+{
+  "itemGroup.paladins.general": "骑士与牧师",
+
+  "item.paladins.paladin_spell_book": "骑士圣契",
+  "item.paladins.priest_spell_book": "神圣书卷",
+
+  "item.paladins.paladin_armor_head": "骑士头盔",
+  "item.paladins.paladin_armor_chest": "骑士胸甲",
+  "item.paladins.paladin_armor_legs": "骑士护腿",
+  "item.paladins.paladin_armor_feet": "骑士靴子",
+  "item.paladins.crusader_armor_head": "十字军头盔",
+  "item.paladins.crusader_armor_chest": "十字军护甲",
+  "item.paladins.crusader_armor_legs": "十字军护腿",
+  "item.paladins.crusader_armor_feet": "十字军靴子",
+
+  "item.paladins.stone_claymore": "石双刃大剑",
+  "item.paladins.iron_claymore": "铁双刃大剑",
+  "item.paladins.golden_claymore": "金双刃大剑",
+  "item.paladins.diamond_claymore": "钻石双刃大剑",
+  "item.paladins.netherite_claymore": "下界合金双刃大剑",
+  "item.paladins.aeternium_claymore": "永恒双刃大剑",
+
+  "item.paladins.wooden_great_hammer": "木巨锤",
+  "item.paladins.stone_great_hammer": "石巨锤",
+  "item.paladins.iron_great_hammer": "铁巨锤",
+  "item.paladins.golden_great_hammer": "金巨锤",
+  "item.paladins.diamond_great_hammer": "钻石巨锤",
+  "item.paladins.netherite_great_hammer": "下界合金巨锤",
+  "item.paladins.aeternium_great_hammer": "永恒巨锤",
+
+  "item.paladins.stone_mace": "石狼牙棒",
+  "item.paladins.iron_mace": "铁狼牙棒",
+  "item.paladins.golden_mace": "金狼牙棒",
+  "item.paladins.diamond_mace": "钻石狼牙棒",
+  "item.paladins.netherite_mace": "下界合金狼牙棒",
+
+  "item.paladins.priest_robe_head": "牧师兜帽领",
+  "item.paladins.priest_robe_chest": "牧师法衣",
+  "item.paladins.priest_robe_legs": "牧师长裤",
+  "item.paladins.priest_robe_feet": "牧师靴子",
+  "item.paladins.prior_robe_head": "修院长兜帽领",
+  "item.paladins.prior_robe_chest": "修院长法衣",
+  "item.paladins.prior_robe_legs": "修院长长裤",
+  "item.paladins.prior_robe_feet": "修院长靴子",
+
+  "item.paladins.acolyte_wand": "侍者魔杖",
+  "item.paladins.holy_wand": "神圣魔杖",
+  "item.paladins.netherite_holy_wand": "下界合金神圣魔杖",
+
+  "item.paladins.holy_staff": "神圣权杖",
+  "item.paladins.netherite_holy_staff": "下界合金神圣权杖",
+  "item.paladins.ruby_holy_staff": "红宝石神圣权杖",
+
+  "spell.paladins.flash_heal.name": "闪耀治疗",
+  "spell.paladins.flash_heal.description": "治疗自身或己方目标{heal}点生命值。",
+  "spell.paladins.divine_protection.name": "神圣保护",
+  "spell.paladins.divine_protection.description": "{effect_duration}秒持续时间内减少50%任何形式伤害。",
+  "effect.paladins.divine_protection": "神圣保护",
+  "spell.paladins.judgement.name": "审判",
+  "spell.paladins.judgement.description": "攻击目标及其{impact_range}方块范围内的敌人, 造成{damage}点伤害并附加眩晕效果，持续{effect_duration}秒。",
+  "effect.paladins.judgement": "眩晕",
+
+  "spell.paladins.holy_shock.name": "神圣冲击",
+  "spell.paladins.holy_shock.description": "治疗自身或己方目标{heal}点生命值，撞击敌人造成{damage}点法术伤害。",
```

```suggestion
  "spell.paladins.holy_shock.description": "治疗自身或己方目标{heal}点生命值，重击敌人造成{damage}点法术伤害。",
```
这么改可能会有点歧义
原版的`亡灵杀手`叫`Smite`，可以测试一下，不过也不推荐往这上面改就是了

---

实际测试效果，确实没有撞击
（虽说是重击但是伤害少得可怜desu）